### PR TITLE
Updated contents

### PR DESCRIPTION
**Pull request by psy2848048**
- Added UI guide
- Updated some messages

### CODE REVIEW

File: content/docs/reference/pair.md [hunk 1]
```diff
@@ -59,6 +60,7 @@ Swap between the given two tokens. It can be considered as trade.<br />
         "belief_price": 0.1,  // optional
         "max_spread": 0.1, // optional
         "to": "<Addr>", // optional
+        "deadlint": 123123 // optional, unix epoch
```
Comment (JoowonYun) [inline content/docs/reference/pair.md:63]: typo

File: content/docs/reference/pair.md [hunk 2]
```diff
@@ -81,6 +83,7 @@ Swap between the given two tokens. It can be considered as trade.<br />
                 "belief_price": 0.1,  // optional
                 "max_spread": 0.1, // optional
                 "to": "<Addr>", // optional
+                "deadlint": 123123 // optional, unix epoch
```
Comment (JoowonYun) [inline content/docs/reference/pair.md:86]: typo

File: content/docs/reference/pair.md (hunk 1 above)
Comment (psy2848048) [reply content/docs/reference/pair.md:63]: [42f63e9](https://github.com/dezswap/docs/pull/9/commits/42f63e9673ddcf42ccfc836552422b161202d77e)

File: content/docs/reference/pair.md (hunk 2 above)
Comment (psy2848048) [reply content/docs/reference/pair.md:86]: [42f63e9](https://github.com/dezswap/docs/pull/9/commits/42f63e9673ddcf42ccfc836552422b161202d77e)

**Review by JoowonYun — APPROVED** [review]

**Review by caramis — APPROVED** [review]
LGTM

**Pull request merged**